Backend/WebAPI/Controllers/ResetController.cs

```diff
@@ -0,0 +1,123 @@
+                    rs.AddResetIDToDB(email, resetID);
+                    string resetURL = rs.CreateResetURL(resetID);
+                    rs.SendResetEmailUserExists(email, resetID);
+                }
+                else //Have a resetID
+                {
+                    //See how many resetIDs the email has
+                    if (rs.GetResetIDCount(email) < 4)
+                    {
+                        //Get the most recent resetID associated with the email provided
+                        string resetIDAlreadyGivenToUser = rs.GetResetID(email);
+                        if (!rs.IsLockedOut(resetIDAlreadyGivenToUser))//Check to see if the resetID isn't locked
+                        {
+                            if (!rs.IsResetIDIsExpired(resetIDAlreadyGivenToUser)) //If the resetID given to the user is expired
+                            {
+                                //Delete the resetID from the DB
+                                rs.DeleteResetIDFromDB(resetIDAlreadyGivenToUser);
+                                //Give the user a new resetID
+                                string resetID = rs.CreateResetID();
+                                rs.AddResetIDToDB(email, resetID);
+                                string resetURL = rs.CreateResetURL(resetID);
+                                rs.SendResetEmailUserExists(email, resetID);
+                            }
+                            else
+                            {
+                                //Tell the user that to check their email
+                            }
+                        }
+                        else
+                        {
+                            //get the expiration date of the token, and add 24 hours to it
+                            //if 24 hours has passed since they failed the security questions
+                                //delete all tokens
+                            //else 
+                                //Tell the user they cannot attempt to reset the password for 24 hours
+                        }
+                    }
+                    else
+                    {
+                        // get the most recent token
+                        //get the expiration time for that token
+                        //add 24 hours to it
+                        //check if that time is after the current time
+                        //if the 24 hours hasn't passed
+
+                            //Tell the user they cannot attempt to reset the password for 24 hours
+                        //if the 24 hours has passed
+                            //delete all tokens
+                    }
+                }
+            }
+            else
+            {
+                rs.SendResetEmailUserDoesNotExist(email);
+            }
+        }
+
+        //After the user clicks the link in the email, this action gets called
+        [HttpPost]
+        [ActionName("ResetPassword")]
```

Comment:
Remove the name from this action. This can just be a POST request to the base URL.